content/pages/blogposts/2024-06-17-hybrid-search.md

```diff
@@ -0,0 +1,9 @@
+Title: Hybrid Search with Apache Solr
+category: solr/blogposts
+slug: hybrid-search-with-apache-solr
+URL: blogposts/hybrid-search-with-apache-solr.html
+save_as: blogposts/hybrid-search-with-apache-solr.html
```

Comment:
If you add a `summary:` tag, it will be used as into text on the blog home page.

Reply:
+1 to Jan's comment.  The `summary` metadata tag will be used as the snippet on the main page (truncated at 512 characters, if it happens to be longer).

Reply:
Please click "Resolve conversation" for all review comments that have been resolved.